### ISSUE TITLE
Non Linearity on Outputted Disparity.

### ISSUE THREAD

**Issue by HimGautam** [mrharicot/monodepth#253]
In the code the disparity map is having non linearity as given below,
```
def get_disp(self, x):
    disp = 0.3 * self.conv(x, 2, 3, 1, tf.nn.sigmoid)
    return disp
```
whereas in the paper non linearity of disparity map is given by scaled sigmoid (0, dmax), where dmax = 0.3× the image
width at a given output scale.
Which one should I use?
